src/pymedphys/_labs/paulking/profile.py

```diff
@@ -207,31 +239,31 @@ def symmetrise(self):
         pass
 
 
-class DoseDepth(Dose1D):  # SHOULD DOSE PROFILE SUPPORT PDD?
-    pass
+class DoseDepth(Dose1D):
+    pass  # SEPARATE CLASS NEEDED?
```

Comment:
If you don't think so, then let's keep it the same.

Reply:
We'll see how it plays out.